tool/csv_writer_gobuster.go

```diff
@@ -0,0 +1,75 @@
+package tool
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func remove(slice []string, s int) []string {
+	return append(slice[:s], slice[s+1:]...)
+}
+
+func lineSkipper(data *os.File) *os.File {
+	r := bufio.NewReader(data)
+	for {
+		_, err := r.ReadBytes('\n')
+		if err == nil {
+			break
+		}
+	}
+	return data
+}
+
+func reformatLine(lineArray []string) []string {
+	regex, _ := regexp.Compile(`\[(.*?)\]|\((.*?)\)`)
```

Comment:
i put the regex in const : (`\[(.*?)\]|\((.*?)\)`)
Because i cannot put this (regex := regexp.MustCompile(`\[(.*?)\]|\((.*?)\)`)) const cause regex cannot be const

Reply:
A var sorry.

Reply:
Set on top of the file 

```go
var regex := regexp.MustCompile(`\[(.*?)\]|\((.*?)\)`)
```